src/subhalo_unbind.cpp

```diff
@@ -481,19 +484,43 @@ void Subhalo_t::RecursiveUnbind(SubhaloList_t &Subhalos, const Snapshot_t &snap)
   ParticleList_t particle_backup;
```

Comment:
Is this still required? Orphans no longer have particles associated to them.

Reply:
I agree with we should remove it